src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,35 @@
+package codeu.controller;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+import java.io.IOException;
+
+public class ProfileServlet extends HttpServlet {
+  // public void doGet(HttpServletRequest request, HttpServletResponse response)
+  //     throws IOException, ServletException {
+  //       response.getOutputStream().println("Hello");
+  // }
+      @Override
+      public void init() throws ServletException {
+          super.init();
+      }
+
+
+      @Override
+      public void doGet(HttpServletRequest request, HttpServletResponse response)
+          throws IOException, ServletException {
+            request.getRequestDispatcher("/WEB-INF/view/profile.jsp").forward(request, response);
+      }
+
+
+
+      @Override
+      public void doPost(HttpServletRequest request, HttpServletResponse response)
+          throws IOException, ServletException {
+            String userValue = request.getParameter("AboutMe");
+            System.out.println(userValue);
```

Comment:
Nice start! Next you will probably want to create a model to store the "AboutMe" in. Usually we wouldn't leave print statements in production code, so here in this PR I might replace this with a comment along the lines of `// TODO: Get the AboutMe off of the request and store it` If you read through the CodeU site, I think adding the AboutMe data is beyond the prototype, so don't worry about getting ahead of yourself too much here. In the next PR you can implement this part :smile: